core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -167,4 +167,12 @@ public boolean isReady() {
    * <p>This method may safely be called concurrently from multiple threads.
    */
   public abstract boolean isCancelled();
+
+  /**
+   * Enables per-message compression, if an encoding type has been negotiated.  If no message
+   * encoding has been negotiated, this is a no-op.
+   */
+  public void setMessageCompression(boolean enabled) {
```

Comment:
We'll probably want to iterate on this name, but it is fine for now (assuming experimental)